hooks/post-command

```diff
@@ -31,6 +33,18 @@ main() {
     exit 0
   fi
 
+  codecov_bash_dir="${DIR}/codecov"
+  mkdir -p "${codecov_bash_dir}"
+  pushd "${codecov_bash_dir}" >/dev/null
+  curl -s -S --connect-timeout 10 --retry 3 --retry-delay 10 -o codecov https://codecov.io/bash
+  curl -s -S --connect-timeout 10 --retry 3 --retry-delay 10 -o env https://codecov.io/env
+  codecov_version=$(grep 'VERSION=\".*\"' codecov | cut -d'"' -f2);
+  if ! curl -s "https://raw.githubusercontent.com/codecov/codecov-bash/${codecov_version}/SHA1SUM" | sha1sum -c - > /dev/null; then
```

Comment:
can we make this a function and verify both `env` and `codecov`?

Reply:
actually, sha1 is verified for both files here, as https://raw.githubusercontent.com/codecov/codecov-bash/1.0.2/SHA1SUM contains sha1 for both `codecov` and `env`.